lib/dialects/mssql/query.js

```diff
@@ -164,8 +164,8 @@ module.exports = (function() {
   Query.prototype.handleShowTablesQuery = function(results) {
     return results.map(function(resultSet) {
       return {
-        tableName: resultSet.TABLE_NAME,
-        schema: resultSet.TABLE_SCHEMA
+        tableName: resultSet.tableName,
+        schema: resultSet.schema
```

Comment:
There shouldn't be a need to do any mapping here, since we are mapping to the same properties as before now ....